indexing-solr/src/test/java/edu/unc/lib/boxc/indexing/solr/test/MockRepositoryObjectHelpers.java

```diff
@@ -89,4 +92,16 @@ public static void addMembers(ContentContainerObject container, ContentObject...
             child.getResource().addProperty(PcdmModels.memberOf, container.getResource());
         }
     }
+
+    public static Resource makeFileResource(String name) {
+        Model model = ModelFactory.createDefaultModel();
+        Resource resc = model.getResource("http://example.com/rest/" + name);
```

Comment:
I'd suggest using `PIDs.get(name).getRepositoryPath()`, it would be able to handle full URLs or UUIDs being passed in, and it'll produce a URI more likely to be consistent with other parts of the code that use PIDs.

Although looking at this class, I'm realizing that we could probably use `makeFileObject` instead to produce a FileObject with a resource. We don't care about the repositoryObjectLoader part, but we could have it skip that one line if the loader is null, or split it up into two methods, one that just takes a pid and does almost all the work, and the other which calls the first method and then does the one repositoryObjectLoader related line.

Reply:
would that affect the other test that use `makeFileObject`? Or does that not matter?

Reply:
nvm i got it!